src/stactools/palsar2_scansar/constants.py

```diff
@@ -40,14 +40,27 @@
     Link(
         rel="documentation",
         target=("https://www.eorc.jaxa.jp/ALOS/en/dataset/palsar2_l22_e.htm"),
-        media_type="application/pdf",
-        title="PALSAR-2 ScanSAR NRB product description",
+        media_type=pystac.MediaType.HTML,
+        title="ALOS-2 PALSAR-2 ScanSAR Products",
+    ),
+    Link(
+        rel="documentation",
+        target=("PALSAR2_CARD4L_L2.2_Geotiff_Format_Description_E_20220715.pdf"),
+        media_type=pystac.MediaType.PDF,
+        title="PALSAR-2 ScanSAR NRB L2.2 Geotiff Format description",
```

Comment:
Didn't catch this the first go around, should "description" in title be uppercase? (same for other asset)